README.md

```diff
@@ -66,23 +74,26 @@ jobs:
             readme_path: index.html
             template: "$newline <h2 class='h2-blog'><a class='a-lightblue' href=$url>$title</a></h2>$newline <br>"  # Do not change the template as it will not render good results!
             gh_token: ${{ secrets.TOKEN }}
-
 ```
 
-*Do star the repository! It gives me motivation to develop more projects like this!*
+_Do star the repository! It gives me motivation to develop more projects like this!_
 
 ## Special Mentions
+
 A special thanks to:
-- [Anurag Hazra](https://github.com/anuraghazra/github-readme-stats) for creating readme stats vercel app which is being used for GitHub Stats Image.
-- [Gautam krishna R](https://github.com/marketplace/actions/blog-post-workflow) GitHub action which makes it possible to update blogpost.
-- [Rajveer Narag](https://github.com/RajveerN01) for helping out with front-end developement and mobile view optimization.
-- [Test Room 7](https://github.com/marketplace/actions/update-files-on-github) GitHub action which enables commit of files to repository
-- [GitHub actions](https://docs.github.com/en/free-pro-team@latest/actions) docs, it's the best guide!
 
-*Create blogs, videos and tag me! I will definately have a look and featue them here!*
+-   [Anurag Hazra](https://github.com/anuraghazra/github-readme-stats) for creating readme stats Vercel app which is being used for GitHub Stats Image.
+-   [Gautam krishna R](https://github.com/marketplace/actions/blog-post-workflow) GitHub action which makes it possible to update blogpost.
+-   [Rajveer Narag](https://github.com/RajveerN01) for helping out with front-end development and mobile view optimization.
+-   [Test Room 7](https://github.com/marketplace/actions/update-files-on-github) GitHub action which enables commit of files to repository
+-   [GitHub actions](https://docs.github.com/en/free-pro-team@latest/actions) docs, it's the best guide!
+
+_Create blogs, videos and tag me! I will definitely have a look and feature them here!_
 
 ## Featured Content
-- [Github Action That Automates Portfolio Generation](https://towardsdatascience.com/github-action-that-automates-portfolio-generation-bc15835862dc) - Medium Blog
+
+-   [Github Action That Automates Portfolio Generation](https://towardsdatascience.com/github-action-that-automates-portfolio-generation-bc15835862dc) - Medium Blog
 
```

Comment:
include the latest commits in this PR so that the changes I made today are reflected here

Reply:
Sure!